docs/source/get_started/install.rst

```diff
@@ -7,6 +7,17 @@ To install HiClass from the Python Package Index (PyPI) simply run:
 
     pip install hiclass
 
+Additionally, it is also possible to install optional packages along. To install optional packages run:
+
+.. code-block:: bash
+
+    pip install hiclass"[<extra_name>]"
+
+:literal:`<extra_name>` can have one of the following options:
+
+- ray: Installs the ray package, which is required for parallel processing support.
+- xai: Installs the shap and xarray packages, which are required for explaining Hiclass predictions.
```

Comment:
```suggestion
- xai: Installs the shap and xarray packages, which are required for explaining HiClass' predictions.
```

Reply:
I have included the exact version in `setup.py` as well.

Resolved by [a0d3f59](https://github.com/scikit-learn-contrib/hiclass/pull/106/commits/a0d3f59e05ce4e90eaeee2be791925b735a0f2b8).

Reply:
Resolved by [ae48bf0](https://github.com/scikit-learn-contrib/hiclass/pull/106/commits/ae48bf02a40d4cfcac8aae4586b1107d529167e3).